File: StoreFinder v1.7.4 Android 3/PHP Scripts/storefinder/models/Photo.php
<?php
 
class Photo
{
	public $photo_id;
    public $photo_url;
    public $thumb_url;
    public $store_id;
    public $created_at;
    public $updated_at;
    public $is_deleted;


    // constructor
    function __construct() 
    {

    }
 
    // destructor
    function __destruct() 
    {
         
    }
}
 
?>

File: Restaurant Locator Android v1.4 2/Restau Android v1.4/RestaurantLocator/drinkea/src/main/java/com/models/Photo.java
package com.models;

import java.io.Serializable;

public class Photo implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = -6422572753453920743L;
	
	public String created_at;
	public int photo_id;
	public String photo_url;
	public int restaurant_id;
	public String thumb_url;
	
}
